lib/video-event-polyfill.js

```diff
@@ -86,9 +86,9 @@ class VideoEventPolyfill extends EventEmitter {
   tick () {
 
     if (this.isLive !== undefined && !this.isLive) {
-      if (this.video.duration && (this.video.duration  - this.video.currentTime < 0.5 ||
+      if (this.video.duration && ((this.video.duration  - this.video.currentTime < 0.5 && this.video.currentTime - this.lastCurrentTime < 0.01) ||
```

Comment:
It's an arbitrary threshold. tick is executeed every 200ms, and browsers update the video's current time at least every ~125ms (that's dependant on browser implementation).
This threshold of 10ms should ensure we detect if the playback has moved